config/environment.js

```diff
@@ -1,6 +1,16 @@
 /*jshint node:true*/
+/* global module */
 'use strict';
 
 module.exports = function(/* environment, appConfig */) {
-  return { };
+  var ENV = {};
```

Comment:
While this was not necessary, it allows to reproduce better a default ember app code-styling (e.g. [environment.js](https://github.com/uniplaces/self-service-frontend/blob/dev/config/environment.js) in the self-service platform).